test/support/live_views.ex

```diff
@@ -269,11 +269,14 @@ defmodule Phoenix.LiveViewTest.ParamCounterLive do
   end
 
   def mount(%{test_pid: pid} = session, socket) do
-    do_mount(session, assign(socket, :test_pid, pid))
+     do_mount(session, assign(socket, :test_pid, pid))
   end
 
   defp do_mount(%{test: %{external_disconnected_redirect: redir}}, socket) do
     %{to: to} = redir
+
+    socket = Map.put(socket, :plug, Phoenix.LiveView.Plug)
```

Comment:
I had to do it because of the test from `params_test.exs:150`, it was testing an external `live_redirect/2` from mount disconnected and it didn't have a plug previously. Not so sure about this 🤔